addons/mail/static/src/js/tours/discuss_channel_tour.js

```diff
@@ -4,8 +4,14 @@ import { registry } from "@web/core/registry";
 import { markup } from "@odoo/owl";
 
 registry.category("web_tour.tours").add("discuss_channel_tour", {
-    url: "/odoo/action-mail.action_discuss",
+    url: "/odoo",
     steps: () => [
+        {
+            trigger: "a[data-menu-xmlid='mail.menu_root_discuss']",
+            content: _t("Open Discuss App"),
+            tooltipPosition: "bottom",
+            run: "click",
+        },
```

Comment:
This doesn't seem to work very well in community, there is an endless loop of "click on discuss app" for me, as we are already on it by default (no home menu).

Reply:
Oh... thank you so much. My bad, I correct it directly